xarray/convert.py

```diff
@@ -31,12 +47,9 @@ def to_cdms2(dataarray):
     # we don't want cdms2 to be a hard dependency
     import cdms2
 
-    def encode(var):
-        return maybe_encode_timedelta(maybe_encode_datetime(var.variable))
-
-    def set_cdms2_attrs(var, attrs):
+    def set_cdms2_attrs(_var, attrs):
```

Comment:
I don't like variable shadowing so I prefer to add the underscore, it's because I have had to deal with some really badly written academic code. Here it's clear enough so can change it back.